cluster-agent/controllers/argoproj.io/namespace_reconciler.go

```diff
@@ -31,6 +31,7 @@ const (
 	namespaceReconcilerInterval = 30 * time.Minute // Interval in Minutes to reconcile workspace/namespace.
 	sleepIntervalsOfBatches     = 1 * time.Second  // Interval in Millisecond between each batch.
 	SecretDbIdentifierKey       = "databaseID"     //Secret label key to point DB entry.
+	waitTimeforRowDelete        = 1 * time.Hour    // Number of hours to wait before deleting DB row
```

Comment:
Since this is actually the time to wait before we delete the Operation CR, think this can be `waitTimeForCRDelete` or  `waitTimeForK8sResourceDelete`, or similar